smooth/optimization/run_optimization.py

```diff
@@ -429,13 +429,13 @@ def mutate(parent, attribute_variation):
         delta = min(delta_min, delta_max)
         # sigma influences spread of random numbers
         # try to keep between min and max of attribute
-        sigma = delta / 3 if delta > 0 else 1
+        sigma = delta / 3.0 if delta > 0 else 1.0
```

Comment:
This shouldn't make a difference in Python 3 and above (see PEP 238) but still gives me a safe feeling :)